app/components/ChartSettings/modules/Metadata.js

```diff
@@ -39,10 +51,13 @@ export default class Metadata extends Component {
   };
 
   render() {
+    if (!this.shouldShowMetadata.subtitle) {
+      delete this.state.subtitle;
+    }
     return (
       <AccordionBlock
         title="Metadata"
-        tooltip="Title, caption, credit"
+        tooltip="Title, subtitle, caption, credit"
```

Comment:
Maybe just change to
```
Title and other metadata fields
```
So we don't have to worry about which ones are enabled or not